config/manager/configmap.yaml

```diff
@@ -15,6 +15,6 @@ metadata:
     app.kubernetes.io/component: kusk-gateway-manager
     app.kubernetes.io/instance: kusk-gateway
     app.kubernetes.io/name: kusk-gateway
-    app.kubernetes.io/version: "v1.2.3"
+    app.kubernetes.io/version: "v1.3.1"
```

Comment:
I think we should change this only when we release. I don't think we're ready for that. Is it correct @jasmingacic ?

Reply:
I believe so

Reply:
OK. Resolving.